packages/drivers/http/src/http.js

```diff
@@ -62,8 +71,7 @@ const getDriver = (store) => {
         ownFetch[method.toLowerCase()] = (url, data, options = {}) => {
           const headers = { ...options.headers }
           let appliedOptions = options
-
-          if (data && ['object', 'array'].includes(typeof data)) {
+          if (!['GET', 'HEAD'].includes(method) && data && ['object', 'array'].includes(typeof data)) {
```

Comment:
it's forbidden to set body with GET or HEAD http method.
GET: https://tools.ietf.org/html/rfc7231#section-4.3.1
HEAD: https://tools.ietf.org/html/rfc7231#section-4.3.2

Reply:
1. Why do you change that in this PR?
2. This is not role to make sure people use the HTTP protocol right

Reply:
yes, but the `options` parameters in after `body` parameter.
and if you use driver with : 
`http('NAME').get('QUERY', {}, { headers: { 'Content-Type': 'text/csv' }})`
and not : `http('NAME').get('QUERY', undefined, { headers: { 'Content-Type': 'text/csv' }})`
Your have an exception .
So in my mind it's better to make this changement .

Reply:
This is not related to this PR, and if you have an exception from fetch because you give a body, this is normal / ok.

You can think about a nicer API in an other PR if you want:
 - `get(url, options)`
 - `post(url, body, options)`

But once again, this is not related to this PR

Reply:
ok, I write this on issue : #315